examples/notebooks/titanic/Titanic.ipynb

```diff
@@ -1111,7 +1111,7 @@
     "val df1 = df\n",
     "    // imputing\n",
     "    .fillNulls { sibsp and parch and age and fare }.perCol { mean() }\n",
-    "    .fillNulls { sex }.withValue(\"female\")\n",
+    "    .fillNulls { sex }.with { \"female\" }\n",
```

Comment:
Is it updated on datalore?

Reply:
no, datalore runs the release version of DF, updating it now would break it

Reply:
Hm, probably not in this case, `with` is available in 0.10.1

Reply:
oh of course XD, you're right

Reply:
done!